### PR TITLE
i18n(es): translate images guide

### PR DESCRIPTION
**Pull request by alfredogonzalezmartinez**
#### What kind of changes does this PR include?

- Translated content

#### Description

Translate `guides/images.md`

### CODE REVIEW

File: src/pages/es/guides/images.md [hunk 1]
```diff
@@ -0,0 +1,228 @@
+---
+layout: ~/layouts/MainLayout.astro
+title: Imágenes
+description: Aprende cómo usar imágenes en Astro.
+i18nReady: true
+setup: |
+  import Since from '../../../components/Since.astro';
+---
+
+Astro proporciona varias formas de usar imágenes en tu sitio, tanto si están almacenadas localmente dentro de tu proyecto, enlazadas remotamente o almacenadas en un CMS o una CDN!
```
Comment (kevinzunigacuellar) [inline src/pages/es/guides/images.md:10]:
```suggestion
Astro proporciona varias formas de usar imágenes en tu proyecto, tanto si están almacenadas localmente, enlazadas remotamente o almacenadas en un CMS o una CDN!
```

File: src/pages/es/guides/images.md [hunk 2]
```diff
@@ -0,0 +1,228 @@
...
+// Imagen local almacenada en public/assets/stars.png
+![Un cielo nocturno estrellado.](/assets/stars.png)
+<img src="/assets/stars.png" alt="Un cielo nocturno estrellado." />
+
+// Imagen remota en otro servidor
+![Astro](https://astro.build/assets/logo.png)
+<img src="https://astro.build/assets/logo.png" width="25" alt="Astro" />
+```
+
+### En componentes de un Framework de UI
```
Comment (kevinzunigacuellar) [inline src/pages/es/guides/images.md:72]:
```suggestion
### En componentes de un framework de UI
```

File: src/pages/es/guides/images.md [hunk 3]
```diff
@@ -0,0 +1,228 @@
...
+
+Después de [instalar la integración](/es/guides/integrations-guide/image/#installation), puedes importar y usar estos dos componentes en cualquier lugar donde puedas usar componentes Astro, incluyendo archivos `.mdx`.
+ 
+:::note
+Los componentes Astro `<Image />` y `<Picture />` no pueden ser usados con imágenes en la carpeta `public/`. Usa sintaxis estándar de HTML o Markdown en lugar de estos componentes.
+:::
+
+### `<Image />`
+
+El [componente `<Image />`](/es/guides/integrations-guide/image/#image-) de Astro permite optimizar una imagen individual y especificar el ancho, el alto y/o la relación de aspecto. Puedes transformar tu imagen a un formato de salida particular, que puede ser usado para evitar comprobar el tipo de archivo de las imágenes remotas.
```
Comment (kevinzunigacuellar) [inline src/pages/es/guides/images.md:123]:
```suggestion
El [componente `<Image />`](/es/guides/integrations-guide/image/#image-) de Astro permite optimizar una imagen individual y especificar el ancho, el alto y/o la relación de aspecto. Puedes transformar tu imagen a un formato particular, que puede ser usado para evitar comprobar el tipo de archivo de las imágenes remotas.
```

File: src/pages/es/guides/images.md [hunk 4]
```diff
@@ -0,0 +1,228 @@
...
+```
+
+### `<Picture /> `
+
+El [componente `<Picture />`](/es/guides/integrations-guide/image/#picture-) de Astro puede ser usado para proporcionar imágenes responsivas en tu sitio, incluyendo imágenes con diferentes tamaños, formatos y disposiciones. Puedes permitir que el navegador elija los tamaños de imágenes apropiados, resoluciones y tipos de archivo basados en factores como el tamaño de la pantalla y la conexión. O, puedes especificar reglas que el navegador debe seguir basadas en media queries.
+
+Este componente es útil para optimizar lo que el usuario ve en diferentes tamaños de pantalla o para el art direction.
+
+:::tip
+Revisa la guía de MDN para más información sobre [imágenes responsivas y art direction](https://developer.mozilla.org/en-US/docs/Learn/HTML/Multimedia_and_embedding/Responsive_images).
```
Comment (kevinzunigacuellar) [inline src/pages/es/guides/images.md:171]:
```suggestion
Revisa la guía de MDN para más información sobre [imágenes responsivas y art direction](https://developer.mozilla.org/es/docs/Learn/HTML/Multimedia_and_embedding/Responsive_images).
```

File: src/pages/es/guides/images.md [hunk 5]
```diff
@@ -0,0 +1,228 @@
...
+<Image src={localImage} aspectRatio="16:9" format="avif" />
+<Image src={imageUrl} height={200} aspectRatio="16:9" format="avif" />
+
+// Las imágenes locales también pueden importarse directamente
+<Image src={import('../assets/local.png')} />
+```
+
+### `<Picture /> `
+
+El [componente `<Picture />`](/es/guides/integrations-guide/image/#picture-) de Astro puede ser usado para proporcionar imágenes responsivas en tu sitio, incluyendo imágenes con diferentes tamaños, formatos y disposiciones. Puedes permitir que el navegador elija los tamaños de imágenes apropiados, resoluciones y tipos de archivo basados en factores como el tamaño de la pantalla y la conexión. O, puedes especificar reglas que el navegador debe seguir basadas en media queries.
```
Comment (kevinzunigacuellar) [inline src/pages/es/guides/images.md:166]:
```suggestion
El [componente `<Picture />`](/es/guides/integrations-guide/image/#picture-) de Astro puede ser usado para proporcionar imágenes adaptables en tu proyecto, incluyendo imágenes con diferentes tamaños, formatos y disposiciones. Puedes permitir que el navegador elija los tamaños de imágenes apropiados, resoluciones y tipos de archivo basados en factores como el tamaño de la pantalla y la conexión. O, puedes especificar reglas que el navegador debe seguir basadas en media queries.
```

File: src/pages/es/guides/images.md [hunk 6]
```diff
@@ -0,0 +1,228 @@
...
+
+:::tip
+Revisa la guía de MDN para más información sobre [imágenes responsivas y art direction](https://developer.mozilla.org/en-US/docs/Learn/HTML/Multimedia_and_embedding/Responsive_images).
+:::
+
+Por defecto, el componente `<Picture />` incluirá los formatos `avif` y `webp` además del formato original de la imagen.
+
+#### Imágenes locales
+
+Los archivos de imágenes locales en la carpeta `src` de tu proyecto pueden ser importados en el frontmatter y pasados directamente al componente `<Picture />`. `src`, `widths`, y `sizes` son propiedades requeridas.
```
Comment (kevinzunigacuellar) [inline src/pages/es/guides/images.md:178]:
```suggestion
Las imágenes locales en la carpeta `src` de tu proyecto pueden ser importados en el frontmatter y pasados directamente al componente `<Picture />`. `src`, `widths`, y `sizes` son propiedades requeridas.
```

File: src/pages/es/guides/images.md [hunk 7]
```diff
@@ -0,0 +1,228 @@
...
+
+<Image src={import('../assets/logo.png')} alt="Astro"/>
+<Image src={rocket} width={300} alt="Cohete acercándose a la luna.">
+<Picture src={rocket} widths=[{200, 400, 800}] sizes="(max-width: 800px) 100vw, 800px" alt="Un cohete despegando." />
+<Picture src={galaxy} widths=[{200, 400, 800}] aspectRatio={16/9} sizes="(max-width: 800px) 100vw, 800px" alt="Espacio exterior." />
+```
+
+## Usar imágenes de un CMS o una CDN
+
+Las CDNs de imágenes funcionan con Astro. Usa su URL como el atributo `src` de una imagen como lo haría al escribir HTML o JSX, o como el atributo `src` de una imagen remota con los componentes `<Image />` y `<Picture />`.
```
Comment (kevinzunigacuellar) [inline src/pages/es/guides/images.md:222]:
```suggestion
Las CDNs de imágenes funcionan con Astro. Usa la URL como el atributo `src` de una imagen como lo haría al escribir HTML o JSX, o como el atributo `src` de una imagen remota con los componentes `<Image />` y `<Picture />`.
```

**Review by kevinzunigacuellar — COMMENTED** [review]
Amazing job! I left a few suggestions ☺️

File: src/pages/es/guides/images.md (hunk 4 above)
Comment (alfredogonzalezmartinez) [reply src/pages/es/guides/images.md:171]:
```suggestion
Revisa la guía de MDN para más información sobre [imágenes adaptables y art direction](https://developer.mozilla.org/en-US/docs/Learn/HTML/Multimedia_and_embedding/Responsive_images).
```

File: src/pages/es/guides/images.md (hunk 6 above)
Comment (alfredogonzalezmartinez) [reply src/pages/es/guides/images.md:178]:
```suggestion
Las imágenes locales en la carpeta `src` de tu proyecto pueden ser importadas en el frontmatter y pasadas directamente al componente `<Picture />`. `src`, `widths`, y `sizes` son propiedades requeridas.
```

File: src/pages/es/guides/images.md [hunk 8]
```diff
@@ -0,0 +1,228 @@
...
+
+### `<Image />`
+
+El [componente `<Image />`](/es/guides/integrations-guide/image/#image-) de Astro permite optimizar una imagen individual y especificar el ancho, el alto y/o la relación de aspecto. Puedes transformar tu imagen a un formato de salida particular, que puede ser usado para evitar comprobar el tipo de archivo de las imágenes remotas.
+
+Este componente es útil para imágenes en las que quieres mantener un tamaño consistente en pantallas, o controlar la calidad de una imagen (ej. logos).
+
+#### Imágenes locales
+
+Los archivos de imágenes en la carpeta origen de tu proyecto pueden ser importados en el frontmatter y pasados directamente al atributo `src` del componente `<Image />`. Todas las demás propiedades son opcionales y se definirán por defecto con las propiedades originales del archivo de imágenes si no se proporcionan.
```
Comment (alfredogonzalezmartinez) [inline src/pages/es/guides/images.md:129]:
```suggestion
Las imágenes en la carpeta origen de tu proyecto pueden ser importadas en el frontmatter y pasadas directamente al atributo `src` del componente `<Image />`. Todas las demás propiedades son opcionales y se definirán por defecto con las propiedades originales del archivo de imágenes si no se proporcionan.
```

**Comment by kevinzunigacuellar** [comment]
LGTM! 🎉

**Comment by Yan-Thomas** [comment]
Great work @alfredogonzalezmartinez, @kevinzunigacuellar, thanks for this!

**Pull request merged**